travis_requirements.txt

```diff
@@ -15,3 +15,4 @@ pyasn1
 httpie
 mysqlclient==1.3.14
 openapi-spec-validator
+mock==3.0.5
```

Comment:
Note - one new testing dependency. `mock` allows me to do some better testing of the TBANS stuff